fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/indexing/IndexingRangeSet.java

```diff
@@ -225,8 +227,15 @@ public static IndexingRangeSet forIndexBuild(@Nonnull FDBRecordStore store, @Non
      * @return a {@code WrappedRangeSet} for scrubbing the index's entries
      */
     @Nonnull
-    public static IndexingRangeSet forScrubbingIndex(@Nonnull FDBRecordStore store, @Nonnull Index index) {
-        RangeSet rangeSet = new RangeSet(IndexingBase.indexScrubIndexRangeSubspace(store, index));
+    public static IndexingRangeSet forScrubbingIndex(@Nonnull FDBRecordStore store, @Nonnull Index index, int rangeId) {
+        final Subspace subspace;
+        if (rangeId == 0) {
+            // Backward compatible
+            subspace = IndexingSubspaces.indexScrubIndexRangeSubspaceLegacy(store, index);
```

Comment:
This uses the word `legacy`, perhaps we should have a comment on the builder that the scrubbing id will be required in the future?

Reply:
Change to `IndexingSubspaces.indexScrubIndexRangeSubspaceZero(..)`.